client/src/components/rounds/Apply.tsx

```diff
@@ -125,7 +127,19 @@ function Apply() {
     };
   }, [props.applicationState]);
 
+  useEffect(() => {
+    if (
+      props.applicationState?.status === ApplicationStatus.Error ||
+      props.applicationError
+    ) {
+      toggleErrorModal(true);
+      toggleStatusModal(false);
+    }
+  }, [props.applicationStatus, props.applicationError]);
+
   if (props.roundStatus === RoundStatus.Error) {
+    // ? not sure if this is the right way to handle this @gravityblast
```

Comment:
@codenamejason I think we can remove this if statement if we already toggle the modal above